src/js/index.js

```diff
@@ -157,11 +158,25 @@ export const hpe = deepFreeze({
       border: {
         radius: '4px',
       },
+      extend: ({ theme }) => `
+        margin-top: ${theme.global.edgeSize.xxsmall};
+      `,
       shadowSize: 'medium',
     },
     elevation: {
+      light: {
```

Comment:
How were these different box shadow sizes calculated?

Reply:
There needs to be a namespace conversation. Figma styles reference 'strong' 'weak' etc, whereas Grommet uses t-shirt sizing. The "calculations" were just done mapping the shadow-default in [Figma](https://www.figma.com/file/eZYR3dtWdb9U90QvJ7p3T9/HPE-Color-Styles?node-id=405%3A25) to 'medium' here, 'weak' to 'small', 'strong' to 'xlarge', then using the logical increments from there. The y-offsets and blur radii are very similar to Grommet (which has inconsistencies between light and dark). The Figma file is consistent across light/dark and Vicky and Greg confirmed.

Reply:
Gotcha, this makes sense. Just hadn't seen them in a file and was curious. Agree that we should have a namespace discussion to align designs with grommet.

Reply:
In the meanwhile, it will be helpful to add a comment here, so it won't block the progress of this PR.

Reply:
comments added.